src/test/java/com/example/amazoncom/BrowserStackHomePage.java

```diff
@@ -0,0 +1,40 @@
+package com.example.amazoncom;
+
+import static org.testng.Assert.assertEquals;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class BrowserStackHomePage {
+  WebDriver driver;
+  @FindBy(xpath = "//input[contains(@id,'twotabsearchtextbox')]")
+  WebElement inputtext;
+  @FindBy(xpath = "//title")
+  WebElement ActualTitle;
+  @FindBy(xpath = "//h1")
+  WebElement Header;
+  @FindBy(xpath = "//a[contains(@data-csa-c-slot-id,\"nav_cs_5\")]")
+  WebElement getOffer;
+  @FindBy(xpath = "//input[@id=\"sp-cc-accept\"]")
+  WebElement acceptcookies;
+
+  public BrowserStackHomePage(WebDriver driver) {
+    this.driver = driver;
+    PageFactory.initElements(driver, this);
+  }
+
+  public void veryStringOffer() {
+    String getStringOffer = getOffer.getText();
+    assertEquals(getStringOffer, "Ofertas");
+  }
+
+  public void clickOnacceptcookies() {
+    acceptcookies.click();
+  }
+
+  public void clickOnGetOffer() {
+    getOffer.click();
+  }
+}
```

Comment:
when you see this red icon this means you need to add an empty line in the end of your code